e2e-tests/smart-update/run

```diff
@@ -5,10 +5,28 @@ set -o xtrace
 
 test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
+
 API='pxc.percona.com/v9-9-9'
-IMAGE_PXC_TO_UPDATE=${IMAGE_PXC}
-PXC_VER=$(echo -n "${IMAGE_PXC_TO_UPDATE}" | $sed -r 's/.*([0-9].[0-9])$/\1/')
-IMAGE_PXC="percona/percona-xtradb-cluster-operator:1.4.0-pxc$PXC_VER"
+TARGET_IMAGE_PXC=${IMAGE_PXC}
+CLUSTER="smart-update"
+CLUSTER_SIZE=3
+PROXY_SIZE=1
+
+if [[ "${TARGET_IMAGE_PXC}" == *"percona-xtradb-cluster-operator"* ]]; then
+    PXC_VER=$(echo -n "${TARGET_IMAGE_PXC}" | $sed -r 's/.*([0-9].[0-9])$/\1/')
+else
+    PXC_VER=$(echo -n "${TARGET_IMAGE_PXC}" | $sed -r 's/.*:([0-9]+\.[0-9]+).*$/\1/')
+fi
+if [[ "${PXC_VER}" == "5.7" ]]; then
+    IMAGE_PXC="percona/percona-xtradb-cluster:5.7.30-31.43"
+elif [[ "${PXC_VER}" == "8.0" ]]; then
+    IMAGE_PXC="percona/percona-xtradb-cluster:8.0.19-10.1"
```

Comment:
Why do we need to hardcode it? We need to have possibility to test images which we pass via test's arguments/options.

Reply:
These images are old version images from which we do upgrade to the images which are selected. We have a custom version service deployed and I have changed this to query the version service to get old images.